src/us/kbase/genbank/test/TestCommandLine.java

```diff
@@ -25,7 +25,7 @@
  * Date: 10/29/14
  * Time: 12:17 AM
  */
-public class CommandLineTest {
+public class TestCommandLine {
```

Comment:
Is this a test driver/runner?  If we're renaming I would suggest TestDriverCommandLine.
I don't see specific tests in here to test the command line functionality itself, just something that can be used as a driver to run tests.